tests/models/test_player.py

```diff
@@ -1,23 +1,19 @@
 """A module for testing player model."""
 import pytest
 from django.contrib.auth.models import User
-
 from mars_api.models import Player
```

Comment:
needs isorting :D

Reply:
So what styling guidelines are we using? This is from PEP 8
![image](https://user-images.githubusercontent.com/34490649/88656829-d099d300-d0c8-11ea-82fa-0e23754f1965.png)